rentals.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :data
+
+  def initialize(data, person, book)
+    @data = data
```

Comment:
- [ ] Instead of data please change the name to date as required to avoid confusion 👍